### DISCUSSION TITLE
output of Electric_field

### DISCUSSION THREAD

**Discussion by mimboch** [epochpic/epoch#229] [Q&A] [answered]
Hello,
I obtained as results of electric field this table, please I want to know what represents the lines of this table in order to plot them.

Could you help me please.

**Comment by TomGoffrey** [comment]
Sorry. I don't understand the question. You have not included any table. What are you trying to plot?

**Comment by mimboch** [comment]
I have obtained the table attached below, I want to know what represents the lines of this table
![mat](https://user-images.githubusercontent.com/46644633/119803669-90b59f00-bed7-11eb-9298-0887f148ac73.png)

**Comment by TomGoffrey** [comment, marked as answer]
That's the x-component of the electric field. Each (i,j)-th element of that table will correspond to the electric field at a point (x,y) in space. So, for example `data.Electric_Field.Ex.data[3,4]` would correspond to the x-component of the electric field at `x[3], y[4]`, where `x` and `y` are your grid arrays.

I would suggest searching for tutorials of how to visualise 2D data in Matlab. A quick search suggests there are lots of good examples online.
